spec/models/token_spec.rb

```diff
@@ -36,7 +36,7 @@
 
   context "token" do
     it "should call JWT.decode" do
-      expect(JWT).to receive(:decode).with(token.send(:jwt_string), token.send(:secret_key), token.send(:hmac_sha_algo))
+      expect(JWT).to receive(:decode).with(token.send(:jwt_string), token.send(:secret_key), true, {:verify_expiration=>false})
```

Comment:
Line is too long. [127/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.